content/en/docs/create-project/_index.md

```diff
@@ -1,9 +1,38 @@
 ---
 title: "Creating projects"
-date: 2017-01-05
 weight: 30
 description: >
-  Now that you've setup the platform, let's create your first project.
+  Now that you've setup the platform, let's work with your projects.
 aliases:
   - /docs/getting-started/first-project/
 ---
+
+Whether you're new to development or have multiple existing projects, Jenkins X can help you automate your building, testing, and deployments quickly and easily!
+
+### Quickstarts
+
+If you're just starting your development process with Jenkins X and want to use some tested, supported project templates that you can setup quickly, you can create a **Quickstart**!
+
+Quickstarts are pre-made applications you can create a project from, instead of making one from scratch.
+
+{{< cards >}}
+  {{< card-body card-title="Node HTTP" card-detail="The HTTP interfaces in Node.js are designed to support many features of the protocol which have been traditionally difficult to use." url="/docs/create-project/creating/" fa-symbol="cube" >}}
+
+  {{< card-body card-title="Spring Boot" card-detail="Spring Boot is Spring's convention-over-configuration solution for creating stand-alone, production-grade Spring Applications." url="/docs/create-project/creating/create-spring" fa-symbol="leaf" >}}
+
+  {{< card-body card-title="Gradle" card-detail="Gradle is an open-source build automation system that leverages Apache Ant and Apache Maven concepts with Groovy-based language for project configuration." url="/docs/create-project/creating/" fa-symbol="hippo" >}}
```

Comment:
To be honest I am a bit confused about what we are gaining here. I can see that Spring and ML have specific targets, but all other cards link to the generic creating page:

![screencapture-jx-docs-jx-jenkins-x-jx-docs-pr-2858-jenkins-x-live-docs-create-project-2020-06-11-10_19_53](https://user-images.githubusercontent.com/296354/84362354-66d37180-abcd-11ea-8953-543d633b0f24.png)

Selecting Gradle or Maven:

![Screenshot 2020-06-11 at 10 21 31](https://user-images.githubusercontent.com/296354/84362392-75ba2400-abcd-11ea-880e-61c93f771e88.png)

Is this not confusing? I would have expected to see some relation to the actual framework I am choosing on the card.

Reply:
BTW, the ML Quickstart page contains content which does not belong there.

![Screenshot 2020-06-11 at 10 32 12](https://user-images.githubusercontent.com/296354/84363450-0b09e800-abcf-11ea-90de-404c7822b55d.png)